docs/jsonapi-spec.md

```diff
@@ -536,14 +536,93 @@ Creates a new collection in the current namespace.
+
+* In high-dimensional vector spaces, such as those produced by embedding algorithms or neural networks, similar items are represented by vectors that are close to each other.
+* The cosine similarity between two vectors is a measure of their directional similarity, regardless of their magnitude. If you compute the dot product of two normalized vectors, you get the cosine similarity.
+
+Thus, by computing the dot product between a query vector and the vectors in the Astra DB database, you can efficiently find items in the database that are directionally similar to the query. With dot_product metric set in the JSON API createCollection command, Astra DB can use the dot product as a measure of similarity between vectors.
+
+##### *euclidean*
+
+When the `createCollection` vector metric is set to `euclidean`, the term refers to the Euclidean distance, which is the most common way of measuring the "ordinary" straight-line distance between two points in Euclidean space.
+
+Given two points P and Q in an n-dimensional space with the following coordinates:
+
+![P and Q coordinates](https://docs.datastax.com/en/astra-serverless/docs/develop/_images/p-q-coordinates.png)
+
+The Euclidean distance between these two points is defined by the following formula:
+
+![Distance formula](https://docs.datastax.com/en/astra-serverless/docs/develop/_images/euclidean-distance-formula.png)
+
+Here’s the formula that calculates how the Euclidean distance (the result of the formula above) is then used to determine the Euclidean similarity value:
+
+![Formula for Euclidean similarity value](https://docs.datastax.com/en/astra-serverless/docs/develop/_images/sim-eucl-from-distance-eucl-formula.png)
+
+For a 2-dimensional space (like a flat plane), it’s the direct straight-line distance between two points. For a 3-dimensional space (like the space we live in), it’s the direct straight-line distance between two points in that space. The formula generalizes this concept to any number of dimensions.
+
+In the context of a vector-enabled Astra DB database:
+
+* **Vectors as Points**: Each vector in the database can be thought of as a point in some high-dimensional space.
+
+* **Distance Between Vectors**: When you want to find how "close" two vectors are, you can use various metrics. The Euclidean distance is one of the most intuitive and commonly used metrics. If two vectors have a small Euclidean distance between them, they are close in the vector space; if they have a large Euclidean distance, they are far apart.
+
+* **Querying and Operations**: When you set "metric": "euclidean" in the JSON API createCollection command, Astra DB can use the Euclidean distance as the metric for any operations that require comparing vectors. For instance, if you’re performing a nearest neighbor search, the Astra DB database will return vectors that have the smallest Euclidean distance to the query vector.
```

Comment:
For consistency, better to say "when you set the vector metric to `euclidean`"

Reply:
Thanks. Done in most recent commit.